client/src/components/Project/index.jsx

```diff
@@ -14,76 +13,147 @@ class Scrums extends Component {
     },
     scrums: [],
     scrumName: '',
-  }
+    error: {
+      status: false,
+      msg: ''
+    }
+  };
 
   componentDidMount() {
-    // fetch
-    const project = {
-      id: 1,
-      projectName: 'Demo'
-    }
-    this.setState({ scrums, project });
+    // fetch project data
+    const { projectId } = this.props.match.params;
+    axios
+      .get(`/api/v1/projects/${projectId}`)
+      .then(result => {
+        const {
+          data: { data },
+          status
+        } = result;
+        if (status === 200) {
+          const project = {
+            id: data.id,
+            projectName: data['name']
+          };
+          this.setState({ project });
+        }
+      })
+      .catch(e => {
+        this.setState({
+          error: {
+            status: true,
+            msg: 'Error loading scrums !!!'
+          }
+        });
+      });
+
+    //fetch scrums
+    axios
+      .get(`/api/v1/projects/${projectId}/scrums`)
+      .then(result => {
+        const {
+          data: { data },
+          status
+        } = result;
+        if (status === 200) {
+          this.setState({ scrums: data });
+        }
+      })
+      .catch(e => {
+        this.setState({
+          error: {
+            status: true,
+            msg: 'Error loading scrums !!!'
+          }
+        });
+      });
   }
 
   handleAddScrum = () => {
-    this.setState((prevState) => {
+    this.setState(prevState => {
       const previousScrums = prevState.scrums;
       if (!previousScrums.length) {
         return {
           scrums: previousScrums.concat({
             id: 1,
-            scrumName: `scrum 1`,
+            scrumName: `scrum 1`
           })
-        }
-      }
-      else {
-        const lastScrumId = previousScrums[previousScrums.length - 1].id
+        };
+      } else {
+        const lastScrumId = previousScrums[previousScrums.length - 1].id;
         return {
           scrums: previousScrums.concat({
             id: lastScrumId + 1,
-            scrumName: `scrum ${lastScrumId + 1}`,
+            scrumName: `scrum ${lastScrumId + 1}`
           })
-        }
+        };
```

Comment:
@AymanAlqoqa If you want to shorten this, you might be able to remove the if-statement and replace it with something like:
```
const newScrumId = previousScrums.length ? (previousScrums[previousScrums.length - 1].id + 1) : 1
return {
  scrums: [...previousScrums, {
    scrumName: `scrum ${ newScrumId }`
    id: newScrumId
  }]
}
```

It's not super important, but then if you had to add another field to new scrums (for example `created_at: new Date()`, then you would only have to add it in one place, not two places.

Reply:
ok, i'll do it if I have time, 
thanks